packages/chord-mark/src/renderer/components/tpl/line.js

```diff
@@ -1,4 +1,8 @@
-const render = ({ line, lineClasses }) => {
-	return `<p class="${lineClasses}">${line}</p>`;
+const render = ({ line, lineClasses, isNewSection, wrapperClasses }) => {
+	if (isNewSection) {
```

Comment:
You can store the wrapping in a variable so there is no need to duplicate the `<p>` rendering part, e.g.:
```javascript
let wrapper = '';
if (isNewSection) {
  wrapper = `</div><div class="${wrapperClasses}">`;
}
return `${wrapper}<p class="${lineClasses}">${line}</p>`;